cetmix_tower_server/wizards/cx_tower_command_execute_wizard.py

```diff
@@ -53,6 +53,7 @@ class CxTowerCommandExecuteWizard(models.TransientModel):
     any_server = fields.Boolean(default=True)
     rendered_code = fields.Text(
         compute="_compute_rendered_code",
+        compute_sudo=True,
```

Comment:
Shouldn't we add `compute_sudo` [here](https://github.com/cetmix/cetmix-tower/blob/3c3f48a4767ca4d83b97913fa3de7914db01aa28/cetmix_tower_server/models/cx_tower_file.py#L119) too?

Reply:
yes, thanks!